numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+# variables.
+seconds_in_minute = 60
+minutes_in_hour = 60
+hours_in_day = 24
+days_in_week = 7
+days_in_year = 365.25
+# my_age = 23.7
+age_20 = 20
+age_100 = 100
+string_1 = "There are "
+string_2 = "That means "
+
+puts string_1 + seconds_in_minute.to_s + " seconds in a minute"
```

Comment:
As a general rule, I like Interpolation better (`#{...}`) because I don't have to remember where to call .to_s.